### ISSUE TITLE
Screen Reader: Many Pages: Emoji button is missing accessible role

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77568]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using VoiceOver on iOS, open the app
2. Swipe right to focus any chat and double-tap to activate it
3. Swipe right to focus the Emoji button and note how it is announced

Other Occurrences:
1. New Chat
2. Group Chat

## Expected Result:
The Emoji button should be properly implemented as a button and announced as "Emoji, button".

## Actual Result:
The Emoji button is missing accessible role and is announced as plain text: "Emoji".

Screen reader users have no way to understand this is an interactable element.

## Area issue was found in:
Chat

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Assign accessibilityRole="button" to the Emoji button to ensure is is announced properly as "Emoji, button".
https://appt.org/en/docs/react-native/samples/accessibility-role
https://reactnative.dev/docs/accessibility#accessibilityrole

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)

## Screenshots/Videos

<img width="1179" height="2438" alt="Image" src="https://github.com/user-attachments/assets/359c706e-9c0b-4c8a-a1c4-202cab4e1d29" />

https://github.com/user-attachments/assets/bbabce1a-0d12-4151-bd06-87257fc112e1

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7009095
**Issue reported by:** NA
**Slack conversation:** NA
